File: plugins/spark_upgrade/tests/resources/spark_conf/input/sample2.scala
package com.piranha;

import org.apache.spark.SparkConf
import org.apache.spark.sql.SparkSession

class Sample {

  def main(argv: Array[String]): Unit = {
        val spark = SparkSession.builder
        .appName("appName")
        .config("conf", "package.conf")
        .enableHiveSupport
        .getOrCreate

        run(spark, config)
  }

}

File: plugins/spark_upgrade/tests/resources/spark_conf/expected/sample2.java
import org.apache.spark.SparkConf;
import org.apache.spark.sql.SparkSession;

public class Sample2 {

  private static JavaSparkContext jsc;

  @BeforeClass
  public static void startSpark() {
    jsc =
        new JavaSparkContext(
            SparkSession.builder()
                .config("spark.sql.legacy.allowUntypedScalaUDF", "true")
                .appName(Sample2.class.getName())
                .master("master")
                .config("spark.driver.allowMultipleContexts", "true")
                .getOrCreate()
                .sparkContext());
  }

}
